tempesta_fw/t/functional/helpers/control.py

```diff
@@ -430,4 +486,22 @@ def servers_get_stats(servers):
     pool = multiprocessing.Pool(threads)
     pool.map(Nginx.get_stats, servers)
 
+def stop_old():
```

Comment:
As we discussed in the chat, that stop should clean up test environment of a given test. Normally that means calling `tearDown()` function. So `./run_tests.sh -C` must take the last test from previous run and call `tearDown()` function of it.

Reply:
we can not run tearDown(), because tearDown() expects, that test structures were initialized before